thor_config.json

```diff
@@ -55,17 +74,36 @@
         "run_next":  null
     },   
 
-    "step6": {  
+    "step8": {  
         "job_name":  "merge-integration-branch-into-stable-and-tag",  
```

Comment:
this step should change to new jenkins job `apply-new-monthly-release-tags-to-all-imgs`